app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,42 @@
+<%= render partial: 'partials/navigation_menu', locals: {visited: 'recipes'} %>
+<main>
+  <h2><%= @recipe.name %></h2>
+  <div>
+    <p><span>Preparation Time:</span> <%= @recipe.preparation_time %></p>
+    <%= render partial: 'switch_button', locals: {public: @recipe.public} %>
+  </div>
+  <div>
+    <p><span>Cooking time:</span> <%= @recipe.cooking_time %>hours</p>
+    <p><span>Description</span> <br/> <%= @recipe.description %></p>
+  </div>
+
+  <div>
+    <%= button_to 'Generate Shopping List', '/general_shopping_list', method: :get %>
+    <% if can? :add_new_food, @recipe %>
+      <%= button_to 'Add Ingredient', '/foods/new', method: :get %>
+    <% end %>
+  </div>
+
+  <hr/>
+  <div class='food-container'>
+    <table class='food-header'>
+      <thead>
+        <tr>Food</tr>
+        <tr>Quantity</tr>
+        <tr>Value</tr>
+        <tr>Actions</tr>
+      </thead>
+      <tbody>
+        <% @foods.each do |food| %>
+          <tr>
+            <td class='food-name'><%= food.name %></td>
+            <td class='food-quantity'><%= food.quantity %> <%= food.measurement_unit %></td>
+            <td class='food-price'>$<%= food.price %></td>
+            <td><div class='delete-food'><%= button_to 'Delete', food_url(food), method: :delete %></div></td>
+          </tr>
+        <% end %>
```

Comment:
- [ ] When I click on the `add ingredient` button, it is directing me to create a food item while it's supposed to direct me somewhere where I could pick up among the food that I have in the system, the ones that make up that recipe. That's the meaning of adding the ingredients. As you can see, you will need to use the relation between recipe and food. That's why there is a recipe_food table 😃 👍  Please work on the feature 🏃